load_average/variables.tf

```diff
@@ -0,0 +1,93 @@
+variable "namespace" {
+  description = "Namespace (e.g. cp or cloudposse)."
+}
+
+variable "stage" {
+  description = "Stage (e.g. prod, dev, staging)."
+}
+
+variable "name" {
+  description = "Name (e.g. bastion or db)."
+}
+
+variable "delimiter" {
+  default = "-"
+}
+
+variable "attributes" {
+  description = "Additional attributes (e.g. policy or role)."
+  type        = "list"
+  default     = []
+}
+
+variable "tags" {
+  description = "Additional tags (e.g. map('BusinessUnit','XYZ')."
+  type        = "map"
+  default     = {}
+}
+
+variable "datadog_api_key" {
+  description = "Datadog API key. This can also be set via the DATADOG_API_KEY environment variable."
+}
+
+variable "datadog_app_key" {
+  description = "Datadog APP key. This can also be set via the DATADOG_APP_KEY environment variable."
+}
+
+variable "monitor_enabled" {
+  description = "State of monitor."
+  default     = true
```

Comment:
enclose in quotes for consistency

Reply:
in this case it will be string

Reply:
![image](https://user-images.githubusercontent.com/52489/31010037-557b515a-a4be-11e7-8bb1-115aa26e4280.png)
https://www.terraform.io/docs/configuration/variables.html#booleans

Reply:
I'm using that variable for  `count = "${var.monitor_enabled}"` , so me need boolean. If `monitor_enabled` will be string we'll get the error `strconv.ParseInt: parsing "false": invalid syntax`.

example:
```terrafrom
variable "bool" {
  default = true
}

variable "string" {
  default = "true"
}

resource "null_resource" "test_bool" {
  count = "${var.bool}"

  provisioner "local-exec" {
    command = "echo BOOL is work"
  }
}

resource "null_resource" "test_string" {
  count = "${var.string}"

  provisioner "local-exec" {
    command = "echo STRING is work"
  }
}
```

```sh
terraform plan
Refreshing Terraform state in-memory prior to plan...
The refreshed state will be used to calculate this plan, but will not be
persisted to local or remote state storage.


------------------------------------------------------------------------
Error running plan: 1 error(s) occurred:

* null_resource.test_string: strconv.ParseInt: parsing "true": invalid syntax

```

Reply:
>  If monitor_enabled will be string we'll get the error strconv.ParseInt: parsing "false": invalid syntax. 

Don't rely on implicit type casting of booleans to `0` and `1`. Instead explicitly calculate the count. See proposed solution above.

Reply:
```
variable "bool" {
  default = true
}

variable "string" {
  default = "true"
}

locals {
  instance_count1 = "${var.bool ? 1 : 0}"
  instance_count2 = "${var.string ? 1 : 0}"
}

resource "null_resource" "test_bool" {
  count = "${local.instance_count1}"

  provisioner "local-exec" {
    command = "echo BOOL worked"
  }
}

resource "null_resource" "test_string" {
  count = "${local.instance_count2}"

  provisioner "local-exec" {
    command = "echo STRING worked"
  }
}
```

this is the right way